conf/test_full.config

```diff
@@ -0,0 +1,36 @@
+/*
+ * -------------------------------------------------
+ *  Nextflow config file for running full-size tests
+ * -------------------------------------------------
+ * Defines bundled input files and everything required
+ * to run a full size pipeline test. Use as follows:
+ *   nextflow run nf-core/bactmap -profile test_full,<docker/singularity>
+ */
+
+params {
+    input = 'https://raw.githubusercontent.com/nf-core/test-datasets/bactmap/samplesheet.csv'
```

Comment:
In the `test_full.config` the idea is to provide as input data full size datasets, e.g. some data that has already been published in SRA / ENA and can be used to test the pipeline with real sized data on AWS.
ftp paths for e.g. SRA data can be directly provided inside a samplesheet.

The results of running the pipeline on this data will be displayed on the AWS results section of the nf-co.re/bactmap page, so people can have a look at representative results of the pipeline.

Reply:
The data in the `test.config` are actually a full test set. Do we need to provide a different test set or can we use the same for both?

Reply:
Ah I guess in this case it is ok to use the same test data